### PR TITLE
copy minuteman_netlink to gen_netlink_client

### PR DESCRIPTION
**Pull request by GoelDeepak**
This is to enable navstar overlay to use gen_netlink

### CODE REVIEW

**Comment by GoelDeepak** [comment]
@sargun

File: rebar.config [hunk 1]
```diff
@@ -3,7 +3,8 @@
 {erl_opts, [debug_info, {parse_transform, lager_transform}]}.
 
 {deps, [
-	{lager, ".*", {git, "git://github.com/basho/lager", {tag, "3.2.1"}}}
+	{lager, ".*", {git, "git://github.com/basho/lager", {tag, "3.2.1"}}},
```
Comment (sargun) [inline rebar.config]: spacing seems weird here. Is it just me?

**Comment by sargun** [comment]
Looks good to me. I wish we were able to add tests in here for the client. We already indirectly have tests in the minueman repo. Do you have any suggestions on how to add tests without doing a ton of work?

File: rebar.config (hunk 1 above)
Comment (GoelDeepak) [reply rebar.config]: It might be tab vs space issue. Will fix it

**Pull request merged**